src/include/stir/Radionuclide.inl

```diff
@@ -0,0 +1,105 @@
+//
+//
+/*!
+
+  \file
+  \brief Implementations of inline functions of class stir::Radionuclide
+
+  \author Daniel Deidda
+  \author Kris Thielemans
+*/
+/*
+    Copyright (C) 2021 National Physical Laboratory
+    Copyright (C) 2021 University College London
+    This file is part of STIR.
+
+    SPDX-License-Identifier: Apache-2.0
+
+    See STIR/LICENSE.txt for details
+*/
+START_NAMESPACE_STIR
+
+std::string
+Radionuclide::parameter_info() const
```

Comment:
I'd prefer this function in a cxx. (constructors and anything that isn't called regularly as well fact, so maybe the whole thing). You'll need to include `<sstream>` etc